build_runner/test/asset/id_test.dart

```diff
@@ -1,71 +0,0 @@
-// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
were the rest of these all dupes?

Reply:
no, the other ones in this directory are testing build_runner stuff.